server/MoveItMoveItServer/Sources/App/GraphQL/GraphQLResolver+Moves.swift

```diff
@@ -20,7 +20,20 @@ struct CreateMoveArguments: Codable {
     let endDate: Date?
 }
 
-class GraphQLResolver {
+protocol MoveResolver {
+    
+    // MARK: - Moves
+    
+    func createMove(request: Request,
+                    arguments: CreateMoveArguments) -> EventLoopFuture<Move>
+    
+    func allMoves(request: Request,
+                  arguments: UserIDArgument) throws -> EventLoopFuture<[Move]>
+    
+    func move(request: Request,
+              arguments: MoveIDArguments) throws -> EventLoopFuture<Move>
+}
```

Comment:
There *should* be an even easier way to do this: Just make `MoveResolver` not have any required methods. Then the default implementations in the extension will just be exposed everywhere that conforms to that protocol. 

Not 100% sure the compiler won't throw a fit without the protocol definitions though